# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../snd-aoa/nodepmod.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.

--- snd-aoa/GNUmakefile.vanilla	2006-04-30 14:01:17.000000000 +0200
+++ snd-aoa/GNUmakefile	2006-05-18 11:31:47.000000000 +0200
@@ -15,7 +15,7 @@
 install: modules
 	install -d $(MODPATH)
 	for mod in `find . -name *.ko` ; do install -m 644 -c $$mod $(MODPATH) ; done
-	/sbin/depmod -a
+	#/sbin/depmod -a
 
 uninstall:
 	rm -rf $(MODPATH)
